apps/api/src/routes/user/me.rs

```diff
@@ -0,0 +1,28 @@
+use crate::routes::{AuthCtx, RspcResult};
+use backend_error_handler::ApiError;
+use backend_prisma_client::prisma::{user, GrammaticalForm};
+use prisma_client_rust;
+use rspc::{internal::specta::Type, selection, ErrorCode};
+use serde::Serialize;
+
+user::include!(user_data { pii_data: select {
+	email
+	pesel
+	birth_date
+	legal_name
+	display_name
+	phone_prefix
+	phone_number
+} }); // TODO create issue for select and include
+
+pub(crate) async fn me(ctx: AuthCtx, _: ()) -> RspcResult<impl Type + Serialize> {
+	let user = ctx
+		.db
+		.user()
+		.find_unique(user::UniqueWhereParam::IdEquals(ctx.user.id))
+		.include(user_data::include())
+		.exec()
+		.await?
+		.ok_or_else(|| ApiError::Rspc(rspc::Error::new(ErrorCode::NotFound, "User not found".into())))?;
+	Ok(selection!(user, {two_factor_auth, grammatical_form, pii_data}))
```

Comment:
You should be able to use `user::select!` to select `two_factor_auth`, `grammatical_form` and the `pii_data` relation. Is that not working and why you mention creating an issue?

Reply:
First of all hi,
It turns out i misunderstood the docs and it caused the select to break and but macro expansion error didn't told me anything then i change it to include and by chance solver error somewhere else and forgot to check if select will work, and left the TODO which was no longer relevant as it turned out after re reading the docs